pkgs/development/python-modules/python-sat/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildPythonPackage, fetchFromGitHub, lib, six, pypblib, pytestCheckHook }:
+
+buildPythonPackage rec {
+	pname = "python-sat";
```

Comment:
Do you differentiate between spaces and tabs in your text editor?

Reply:
For me, on Neovim, there's:

```vim
set listchars=tab:›\ ,trail:-,extends:»,precedes:«,eol:¬
```

Reply:
Thank. Never took care about this configuration in my editor. I've added the editorconfig plugin and enabled non visible character display to my editor.